tests/testthat/helper-demo.R

```diff
@@ -21,134 +20,3 @@ cleanup_tasks_demo <- function(dirinfo) {
   setwd(dirinfo$wd)
   unlink(dirinfo$tmpdir, recursive=TRUE)
 }
-
```

Comment:
The whole set of functions that follows were for an old version of scipiper (an exploration of using `make` instead of `remake`, in fact).